index.html

```diff
@@ -0,0 +1,41 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+   <link href="https://fonts.googleapis.com/css?family=Roboto:100,100italic,300,300italic,regular,italic,500,500italic,700,700italic,900,900italic" rel="stylesheet" />
+   <link href="https://fonts.googleapis.com/css?family=Poppins:100,100italic,200,200italic,300,300italic,regular,italic,500,500italic,600,600italic,700,700italic,800,800italic,900,900italic" rel="stylesheet" />
+   <link href="https://fonts.googleapis.com/css?family=Crete+Round:regular,italic" rel="stylesheet" />
+    <script src="https://kit.fontawesome.com/e971436820.js" crossorigin="anonymous"></script>
+    <title>Document</title>
+</head>
+<body>
+    <header class="main-section">
+        <section class="toolbar">
+            <h3>John Doe</h3>
+            <i class="fa fa-bars" aria-hidden="true"></i>
+        </section>
+        <section class="intro">
+            <div class="name">
+                <h1>Hey There.<br>I'm Michael</h1>
+            </div>
+            <div class="content">
+                <h3>I’am a Software Developer</h3>
+                <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hesitate to contact me.</p>
+            </div>
+            <ul class="socials">
+                <li><a href=""><i class="fab fa-github"></i></a></li>
+                <li><a href=""><i class="fab fa-linkedin-in"></i></a></li>
+                <li><a href=""><i class="fab fa-angellist"></i></a></li>
+                <li><a href=""><i class="fab fa-twitter"></i></a></li>
+                <li><a href=""><i class="fab fa-medium-m"></i></a></li>
+            </ul>
+        </section>
+    </header>
```

Comment:
- [ ] Please, consider, that you have to create two sections: toolbar (header) and headline section. In `body` tag should be two sections. Considering your code, you insert all inside of `header` tag. Please, add your toolbar (first section) into `header` tag and your 2nd headline section (which should be right after the header) into `section` tag, aka you should use HTML5 semantic tags.